content/cybersecurity/concepts/social-engineering/social-engineering.md

```diff
@@ -0,0 +1,25 @@
+---
+Title: 'Social Engineering'
+Description: 'Social engineering is whne an attacker uses human emotion (usually fear and urgency) to trick the target into performing an action, such as send the attacker money, divulge sensitive customer information, or disclose authentication credentials.'
+Subjects:
+  - 'Cybersecurity'
+Tags:
+  - 'Cybersecurity'
+  - 'Hacking'
+CatalogContent:
+  - 'paths/fundamentals-of-cybersecurity'
+  - 'introduction-to-personal-digital-security'
+---
+
+**Social Engineering** is when an attacker uses human emotion (usually fear and urgency) to trick the target into performing an action, such as send the attacker money, divulge sensitive customer information, or disclose authentication credentials.
+
+## Social Engineering In-Use
```

Comment:
Can we exchange this header for what kind of attack this is or name it in the section and link out to: https://www.codecademy.com/resources/docs/cybersecurity/cyber-attack

Reply:
I don't fully understand this ask. Can you elaborate?

Reply:
Please disregard this comment!